pkg/pillar/hypervisor/xen.go

```diff
@@ -155,7 +155,7 @@ func (ctx xenContext) CreateDomConfig(domainName string, config types.DomainConf
 					config.VncDisplay))
 			}
 			if config.VncPasswd != "" {
-				vncParams = append(vncParams, fmt.Sprintf("vncpasswd=\"%s\"\n",
```

Comment:
actually, wait -- I think dropping '\n' here would be a mistake

Reply:
We wrap this lines into vfb=[] here https://github.com/lf-edge/eve/blob/fd0396bd2b2b5faf091bcf19856840dd153fe398/pkg/pillar/hypervisor/xen.go#L161 . So, we need one line.

Reply:
aha! missed that -- sorry for the noise @giggsoff